commit 45193362a810024ab4810470fcd00e16123d5aaf
Author: Richard Braun <rbraun@sceen.net>
Date:   Wed May 22 21:00:08 2013 +0200

    Fix some errors in the texinfo documentation
    
    * doc/gpl.texinfo: Replace @unnumbered with @node and @section commands.
    * doc/hurd.texi: Fix call to @setchapternewpage, remove the @node and
    @section commands that are now in gpl.texinfo, and remove an erroneous
    occurrence of 'attributes' before a command.

diff --git a/doc/gpl.texinfo b/doc/gpl.texinfo
index a2f8db9..d29870c 100644
--- a/doc/gpl.texinfo
+++ b/doc/gpl.texinfo
@@ -1,6 +1,5 @@
-@setfilename gpl.info
-
-@unnumbered GNU GENERAL PUBLIC LICENSE
+@node Copying
+@section GNU GENERAL PUBLIC LICENSE
 @center Version 2, June 1991
 
 @display
diff --git a/doc/hurd.texi b/doc/hurd.texi
index ea73a4c..070daed 100644
--- a/doc/hurd.texi
+++ b/doc/hurd.texi
@@ -127,7 +127,7 @@ into another language, under the above conditions for modified versions.
 @end quotation
 @end copying
 
-@setchapternewpage none
+@setchapternewpage off
 @settitle Hurd Reference Manual
 
 @titlepage
@@ -513,9 +513,6 @@ mentions that the FSF started developing the Hurd in 1990.  As of
 kernel.
 
 
-@node Copying
-@section GNU General Public License
-
 @include gpl.texinfo
 
 
@@ -1461,7 +1458,7 @@ dropped by the kernel.  If the page is currently in core, the kernel
 might ignore this call.
 @end deftypefun
 
-attributes@deftypefun void pager_change_attributes (@w{struct pager *@var{pager}}, @w{boolean_t @var{may_cache}}, @w{memory_object_copy_strategy_t @var{copy_strategy}}, @w{int @var{wait}})
+@deftypefun void pager_change_attributes (@w{struct pager *@var{pager}}, @w{boolean_t @var{may_cache}}, @w{memory_object_copy_strategy_t @var{copy_strategy}}, @w{int @var{wait}})
 Change the attributes of the memory object underlying pager @var{pager}.
 The @var{may_cache} and @var{copy_strategy} arguments are as for
 @code{memory_object_change_}.  Wait for the kernel to report
